WordPress/Classes/Utility/WPTableViewHandler.m

```diff
@@ -205,12 +205,12 @@ - (void)refreshTableViewPreservingOffset
 
 - (void)cacheRowHeight:(CGFloat)height forIndexPath:(NSIndexPath *)indexPath
 {
-    [self.cachedRowHeights setObject:@(height) forKey:[indexPath toString]];
+    [self.cachedRowHeights setObject:@(height) forKey:indexPath];
```

Comment:
@koke This extension was introduced in [this commit](https://github.com/wordpress-mobile/WordPress-iOS/commit/bf2fcdd3a35fcfed3a1b32433b9134f435859905) ("`Notifications: Fixes a row height caching bug`").

Can't remember exactly *what* delegate method was producing this, but bottom line was: UIKit was passing on an internal NSIndexPath subclass, and the equality operator was failing on us. 

I've tried triggering the original bug, but odds are, it was either fixed, or... i can't remember the exact conditions.

Just verified, and cannot trigger the initial conditions. Good to go on my end!

P.s.: Hashable / Equatable are for the Swift struct realm!